server/handler.go

```diff
@@ -581,6 +583,22 @@ func parseQueryValueAsBool(r *http.Request, key string) bool {
 	return b
 }
 
+func parseQueryValueAsInt64(r *http.Request, key string, defaultValue int64) int64 {
```

Comment:
If we don't need to parse other int64 values, you can use a dedicated MaxResults function. e.g.) `parseMaxResultValue`

Reply:
It seems likely that there may be other int64 values that we need to parse in the future, so it seems fine to leave as is

Reply:
Then I think it's ok to leave the function, just do the maxResults's validation somewhere else.